ms-windows/README.md

```diff
@@ -0,0 +1,132 @@
+OpenSSL Installer Build Tools For Windows
+=========================================
+
+This packager prepares binary builds of OpenSSL for the Microsoft Windows operating system and encapsulates the generated files within EXE and MSI installers suitable for distribution.
+
+Basic Setup
+-----------
+
+To perform builds efficiently, a relatively recent desktop class Intel/AMD CPU with at least 8 performance cores is strongly recommended to build all supported versions and architectures simultaneously.  At least 15GB of free disk storage should be available as well.
+
+On a Windows computer (or virtual machine), install Visual C++ components from both Microsoft Visual Studio 2017 and Microsoft Visual Studio 2019.  During the installation process, be sure to select "MSVC ... ARM64" from the optional items under "Desktop development with C++" to be able to build for arm64.  Also be sure to install at least one Windows SDK.
+
+The scripts here are written in the PHP scripting language.  Obtain a "Thread Safe" build of PHP for Windows from [windows.php.net](https://windows.php.net/download/).  The 'zlib', 'zip', and 'openssl' extensions need to be enabled.
+
+Obtain a copy of this repository.  Avoid putting the repository into a directory structure with paths that have spaces in them.  Some tools have difficulties with spaces.  The various paths also get quite long, so the closer to the root of a drive that this repository sits on the machine, the less likely that limitations in Windows itself will be encountered.
+
+Running Build Tools
+-------------------
+
+The main 'build-tools.php' script performs all of the work to download required dependencies (Perl, NASM, Inno Setup, Wix toolset) from HTTPS enabled websites (hence why the 'openssl' extension in PHP is required to build OpenSSL), validate environments, download, validate and extract OpenSSL release tarballs, run `perl Configure` and `nmake`, and run Inno Setup and Wix toolset.
+
+Avoid moving or renaming parent directories after running the build tools script.  It is easier to just start from a completely fresh copy of this repository if you need to move or rename parent folders.  A number of paths get mapped to the current directory structure during the initialization/preparation and build process and will probably result in errors if the parent directories are moved or renamed.
+
+From a Command Prompt, run:
```

Comment:
Being in the 'ms-windows' directory is convenient but not necessary to run the build-tools script.  It figures out where it is on the system and adjusts its own paths accordingly.

The parent directory issue is because paths get hardcoded by the tool once the tool starts being used.  It's just easier to start from a freshly cloned repo in that case than to try to find and replace all the paths.